galaxy/Dockerfile

```diff
@@ -141,7 +141,11 @@ RUN curl -s -L https://repo.continuum.io/miniconda/Miniconda2-4.5.12-Linux-x86_6
 RUN cp $GALAXY_HOME/.bashrc ~/
 RUN mkdir $GALAXY_ROOT \
     && curl -L -s $GALAXY_REPO/archive/$GALAXY_RELEASE.tar.gz | tar xzf - --strip-components=1 -C $GALAXY_ROOT \
-    && PATH=$GALAXY_CONDA_PREFIX/bin/:$PATH virtualenv $GALAXY_VIRTUAL_ENV \
+    && . ~/.bashrc \
```

Comment:
@mvdbeek This line activates the conda environment via https://github.com/bgruening/docker-galaxy-stable/pull/517/files#diff-4f06736bd1b2ae4d3c1684eb69b4e745R125. Yet it still will not function without the other changes. Conda does not set LD_LIBRARY_PATH (and they actively oppose doing so) and virtualenv still fails to link libpython.

Reply:
Well, that's one line too late then.

Reply:
Agreed, I think virtualenv is expecting a version of python compiled statically linked.